package-lock.json

```diff
@@ -296,25 +296,12 @@
             }
         },
         "node_modules/@types/node-fetch": {
-            "version": "2.6.3",
-            "resolved": "https://registry.npmjs.org/@types/node-fetch/-/node-fetch-2.6.3.tgz",
-            "integrity": "sha512-ETTL1mOEdq/sxUtgtOhKjyB2Irra4cjxksvcMUR5Zr4n+PxVhsCD9WS46oPbHL3et9Zde7CNRr+WUNlcHvsX+w==",
+            "version": "2.6.11",
+            "resolved": "https://deagxartifactory.sickcn.net:443/artifactory/api/npm/npm/@types/node-fetch/-/node-fetch-2.6.11.tgz",
```

Comment:
This should be the standard NPM registry, not some random URL registry.

Reply:
Fixed! :)